tests/test_stream.py

```diff
@@ -47,10 +76,10 @@ def test_create_stream(chain, payer, tokens, stream_manager, extra_args):
     with pytest.raises(apepay_exc.StreamLifeInsufficient):
         stream_manager.create(tokens[0], amount_per_second + 1, sender=payer)
 
-    start_time = chain.pending_timestamp
     stream = stream_manager.create(
         tokens[0], amount_per_second, **extra_args, sender=payer
     )
+    start_time = chain.blocks.head.timestamp
```

Comment:
you should use `pytest.approx()` perhaps.. this test sometimes fails for me.

Reply:
I was actually having issues with how I was doing it before, but this should actually be deterministic because it's the last _mined_ block's timestamp, e.g. the block containing the transaction I just made above. I finally realized this was a much better way to do it vs. tracking `chain.pending_timestamp` prior to making the transaction, since transactions take longer to process in CI